RecoTracker/LST/interface/LSTOutput.h

```diff
@@ -0,0 +1,53 @@
+#ifndef RecoTracker_LST_LSTOutput_h
+#define RecoTracker_LST_LSTOutput_h
+
+#include <memory>
+#include <vector>
+
+class LSTOutput {
+public:
+  LSTOutput();
+  ~LSTOutput();
+
+  LSTOutput(LSTOutput const&) = delete;
+  LSTOutput& operator=(LSTOutput const&) = delete;
+  LSTOutput(LSTOutput&&);
+  LSTOutput& operator=(LSTOutput&&);
+
+  void setLSTOutputTraits(std::vector<std::vector<int>> lstTrackCandidate_hitIdx,
+                          std::vector<int> lstTrackCandidate_len,
+                          std::vector<float> lstTrackCandidate_pt,
+                          std::vector<float> lstTrackCandidate_eta,
+                          std::vector<float> lstTrackCandidate_phi) {
+    lstTrackCandidate_hitIdx_ = lstTrackCandidate_hitIdx;
```

Comment:
the repeat of `lstTrackCandidate` seems excessive.
If the idea is to have more kinds of outputs in `LSTOutput`, perhaps make structs for groups of data and pass that around.

This refers to the access methods as well.